CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 
 ## Unreleased
 
+### Deprecation notice
+
+- `getAccessRequestFromRedirectUrl` and `getAccessGrantFromRedirectUrl` no longer
+  pick up VC passed by value, and only support passing by IRI. This is for security
+  reasons, as passing a VC by value in the IR leaks information.
```

Comment:
"in the IR"? Is this a typo of IRI?

Reply:
Yup. Fixed in ee9a9aae3a7c0187c24dc2b0f9d9e7b970e145e8